### PR TITLE
Pedia from mainmenu

### LINKED ISSUE #5432: Add possibility to read Civilopedia from the main menu
**Is your feature request related to a problem? Please describe.**
It would be great if it will be possible to open and read Civilopedia from the main menu. At the moment it is possible only after loading or starting any game.

**Describe the solution you'd like**
Add the little round button to the lower right corner of the main menu with big "C". Or just little standard button with "Civilopedia" text.

### PR DESCRIPTION
**Pull request by SomeTroglodyte**
I allowed Civilopedia access from Map Editor :see_no_evil:  - and there were a few silly crashes due to uncouth expectations on context, like UncivGame.Current.worldScreen being accessible...

<details><summary>Screenies...</summary>

image
image
</details>

Linking button broken - so let's try "Closes #5432"

### CODE REVIEW

**Comment by SomeTroglodyte** [comment]
Sorry, the second half was meant to be a separate PR, forgot a step. Rollback and separate or take as is?

Second half adds Eras category, inspired by #6785 , with tech list and objects linked via `OnlyAvailableWhen` or `PolicyBranch.era`. Screenie in the issue is an early version missing the Policies, but you get the idea. Me making git mistakes means me go pause oogah oogah :orangutan: 

Rex icon for "Eras" header may not please, open for suggestions (I wanted the Rex _skeleton_, too cute, but it was too filigrane - could go for one of the excellent Rex skulls or even a simple "Calendar" symbol).

**Comment by OptimizedForDensity** [comment]
The hour glass icon might work since it symbolizes time and is already in the game.

**Comment by SomeTroglodyte** [comment]
That I used only in testing. Doesn't hold up to say the Wonders or Religions icons i thought - though a lot of the others could use ideas. Simpler more stylized - guide the user without distracting /vs/ more unique and decorative - good for headers? Something like that. Also - running sand = short, and I wanted looooooooong time. Thus the paleontology idea. My favourite of all is the one I gimped for map editor, the terrain one.

File: android/assets/jsons/translations/template.properties [hunk 1]
```diff
@@ -1254,8 +1254,13 @@ Granted by: =
...
 Difficulty levels = 
 The possible rewards are: = 
+Eras = 
+Embarked strength: [amount]† = 
+Base unit buy cost: [amount]¤ = 
+Research agreement cost: [amount]¤ = 
 
-# Policies
+
+S# Policies
```
Comment (xlenstra) [inline android/assets/jsons/translations/template.properties:1263]: S?

**Review by xlenstra — APPROVED** [review]

File: core/src/com/unciv/MainMenuScreen.kt [hunk 2]
```diff
@@ -222,6 +238,13 @@ class MainMenuScreen: BaseScreen() {
         }
     }
 
+    private fun openCivilopedia() {
+        val ruleset =RulesetCache[game.settings.lastGameSetup?.gameParameters?.baseRuleset]
+            ?: RulesetCache[BaseRuleset.Civ_V_GnK.fullName]
+            ?: return
```
Comment (xlenstra) [inline core/src/com/unciv/MainMenuScreen.kt:244]: An alternative could be to open a mod selection screen, where you can select a base ruleset mod and extension mods.

**Comment by yairm210** [comment]
Looks OK to me, can be improved in the future but let's get the base out now :)

File: core/src/com/unciv/MainMenuScreen.kt (hunk 2 above)
Comment (SomeTroglodyte) [reply core/src/com/unciv/MainMenuScreen.kt:244]: As was suggested in one of the issues. Honestly, much to do and I think players will get used to it quickly and everybody has their favourite very likely selected in there. Ruleset selection - may be _in_ pedia, hidden when in-game. More important would be to check the pedia map editor loads shows the correct ruleset - don't remember what it does :grimacing:

File: android/assets/jsons/translations/template.properties (hunk 1 above)
Comment (SomeTroglodyte) [reply android/assets/jsons/translations/template.properties:1263]: Oops, shrapnel typo. Good thing it's neutral - `#` is not the thing marking a comment line in translations, lack of `=` is.

**Pull request merged**